server/middlewares/passport.js

```diff
@@ -0,0 +1,54 @@
+const CONFIG = require('config')
+const __ = CONFIG.universalPath
+const _ = __.require('lib', 'utils')
+const session = require('express-session')
+const passport = require('passport')
+const User = __.require('controllers', '/users/lib/users')
+const gitlabStrategy = require('./strategies/gitlab')
```

Comment:
Was there any specific reason why we're not using https://github.com/fh1ch/passport-gitlab2/blob/master/lib/strategy.js?